cpu/atmega_common/include/getopt.h

```diff
@@ -0,0 +1,75 @@
+/*
+* Copyright (C) 2018 Viraj Sahai <vsahai@usc.edu, virajsahai32@gmail.com>
+*
+* This file is subject to the terms and conditions of the GNU Lesser General
+* Public License v2.1. See the file LICENSE in the top level directory for more
+* details.
+*/
+
+/**
+* @ingroup     cpu_atmega_common
+* @{
+*
+* @file        header
+* @brief       Implementation of getopt lib.
+*
+* This implements the getopt and getopt_r functionality. getopt_r has a few caveats, make sure to use it correctly.
+*
+* @author      Viraj Sahai <vsahai@usc.edu, virajsahai32@gmail.com>
+*
+*/
+
+#ifndef GETOPT_H
+#define GETOPT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct{
```

Comment:
This struct is not documented.

Reply:
I am sorry for being extra lost but I am fairly new to Open Source. I went through all the dev recommendations but didn't find the location where to document it?